CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Remove `generate_extended_key`, returned ExtendedKeyInfo [#154]
   - Remove `restore_extended_key`, returned ExtendedKeyInfo [#154]
   - Remove dictionary `ExtendedKeyInfo {mnenonic, xprv, fingerprint}` [#154]
+  - Remove interface `Transaction` [#190]
+  - Changed `Wallet` interface `list_transaction()` to return array of `TransactionDetail` [#190]
```

Comment:
Small nit: the struct is `TransactionDetails`

Reply:
good catch, fixed. Also rebased.